Dockerfile

```diff
@@ -1,6 +1,10 @@
 ARG ELIXIR_VERSION=1.11.4
 ARG SOURCE_COMMIT
 
+FROM haskell:8.10.4-buster as haskell-builder
+COPY simformat /simformat
+RUN cd simformat && stack install simformat --local-bin-path=/usr/local/bin
```

Comment:
one small thing: since we want to eventually [upstream the local merge changes](https://github.com/bors-ng/bors-ng/issues/512) + any other general enhancements we end up making to bors, it would be good to tag these SimSpace-specific additions to the repo so they're easy to leave out later.

something like an easily-greppable comment (concat a uuid with "SIMSPACE"?) added before all the changes in this PR, plus a small note in a file at the root of the repo `SIMSPACE.md` about the comment tag would work

Reply:
I marked the changes with comments, what do you think?